include/vigra/mathutil.hxx

```diff
@@ -164,6 +164,17 @@ VIGRA_DEFINE_MISSING_ABS(signed long long)
 using std::isinf;
 using std::isnan;
 
+#ifdef _LIBCPP_VERSION
+// On older versions of Mac, the system's libc++ does not provide isnan(double)
+// So we just provide it ourselves.
```

Comment:
Oooops!  The comment here is incorrect.  All versions of `libc++` provide `isnan(double)`.  It is the integer types that cause the problem, and that is what is fixed here.